core/src/main/scala/kafka/tools/StorageTool.scala

```diff
@@ -313,22 +347,22 @@ object StorageTool extends Logging {
     }
 
     def getSaltedPassword(
-      argMap: Map[String,String],
-      scramMechanism : ScramMechanism,
-      salt : Array[Byte],
-      iterations: Int
-    ) : Array[Byte] = {
+                           argMap: Map[String,String],
+                           scramMechanism : ScramMechanism,
+                           salt : Array[Byte],
+                           iterations: Int
+                         ) : Array[Byte] = {
```

Comment:
Please preserve the old indentation. In Kafka that is how we indent parameters to a method that span multiple lines.